recipes/stac-check/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "stac-check" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stac_check-{{ version }}.tar.gz
+  sha256: de8e053ef8a2ba3a3264c03eb9c573b46564d25a6300d28d0e2e284a8d570250
+
+build:
+  entry_points:
+    - stac-check=stac_check.cli:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - click >=7.1.2
+    - jsonschema >=3.1.2b0
+    - pystac >=1.1.0
+    - pytest
+    - python >=3.7
+    - python-dotenv
+    - pyyaml
+    - requests >=2.19.1
+    - stac-validator >=3.0.0
+
+test:
+  imports:
+    - stac_check
+  commands:
+    - pip check
+    - stac-check --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/stac-utils/stac-check
+  summary: Linting and validation tool for STAC assets
+  license: MIT
+  license_file: LICENSE
+  doc_url: https://github.com/stac-utils/stac-check
+  dev_url: https://github.com/stac-utils/stac-check
+
+extra:
+  recipe-maintainers:
+    - gadomski
```

Comment:
This name needs to match your username or we need to ask @gadmoski if they want to maintain.

Reply:
@jonhealy1 needs to comment themselves here before we can merge

Reply:
Hi